test/pylib/runner.py

```diff
@@ -186,7 +186,11 @@ def pytest_sessionstart(session: pytest.Session) -> None:
 
 def pytest_sessionfinish() -> None:
     if getattr(TestSuite, "artifacts", None) is not None:
-        asyncio.get_event_loop().run_until_complete(TestSuite.artifacts.cleanup_before_exit())
+        loop: AbstractEventLoop = asyncio.get_event_loop()
```

Comment:
I don't get how this change is connected with "add different levels of verbosity for output" description. Please, explain

Reply:
done